client/test/unit/specs/transactions.spec.js

```diff
@@ -75,8 +75,8 @@ describe('Testing Transactions.vue', () => {
         consentFlag: null,
         consent_dt: null,
         corpNum: null,
-        eventDate: 'Thu, 15 Jul 2021 21:36:03 GMT',
-        expirationDate: '2021-09-11T07:01:00+00:00',
+        eventDate: 'Thu, 07 Oct 2021 21:36:03 GMT',
+        expirationDate: '2021-12-01T07:59:00+00:00',
```

Comment:
This is incorrect. In December, the UTC offset is 7h (11:59 Pac + 7h = 06:59 UTC).

Reply:
fixed